jsapp/scss/components/_kobo.navigation.scss

```diff
@@ -136,6 +136,7 @@ $headerAccountTextColor: #dbedf7;
 .popover-menu--account-menu {
   .popover-menu__content {
     min-width: 240px;
+    z-index: 1001;
```

Comment:
Would it be easy to create zindexes.scss file for keeping all important `z-indexes` as variables there? We wouldn't need those magical numbers then :)

Reply:
Sure, but let's note it for future work. It's beyond this PR's scope.

Reply:
Noted it down here: https://github.com/orgs/kobotoolbox/projects/6#card-8762792